LIMS_IMAGE/web/templates/orders/shopping.html

```diff
@@ -2,30 +2,85 @@
 <!-- Not having this here makes the interpreter mad -->
 {% block title %}Client Home{% endblock %}
 <!-- Not having this here makes the interpreter mad -->
+<body>  
+    
+    {% block content %}
+
+    <h1>Shopping Cart (Order Submission)</h1> 
+        <br>
+      <p><b> Hello, {{user}} {{account_number}}<br>
+            <br>
+            &emsp; Please complete the order form by selecting your sample type, test type and the quantity for each order. <br>
+            &emsp; If you are not sure which test or package to order please see Appendix B for our recommendation. 
+      </b> </p>   
+        {% if user %} <!--this should be account_number-->
+            <form>
+            <h3> &emsp; Sample submission form</h3>
+            
+                <div class = "container mt-3"> 
+                    <form action = "/action_page.php">
+                        <label for="sample-type" class="form-label">Choose your sample from the list:</label>
+                        <input class="form-control" list="sample-types" name="sample-type" id="sample-type">
+                        <datalist id="sample-types">
+                            <option value="Dairy">
+                            <option value="Cosmetics">
+                            <option value="Environment">
+                        </datalist>   
+                    </form>
+                </div>
 
-<body>
+                <div class = "container mt-3">
+                    <form action = "/action_page.php">
+                        <label for="sample-num" class="form-label"> How many samples of your selected type above you can submit for the testing? </label>
+                        <input class="sample-num" list="sample-nums" name="sample-num" id="sample-num">
+                        <datalist id="sample-nums">
+                            <option>1</option> 
+                            <option>2</option>
+                            <option>3</option>
+                            <option>4+</option>
+                        </datalist>   
+                    </form>
+                </div>
     
-    {% block content%}
-    <h1>Shopping</h1> 
-    <h2>Your shopping cart</h2>
+                <div class = "container mt-3">
+                    <form action = "/action_page.php">
+                        <label for="Packaged-individual" class="form-label">Choose your package or individual test</label>
+                        <p>&emsp; * Please see the appendix B for the recommendation. &emsp; 
+                            <a href = 'appendix-b.html'> Appendix B </a>></p> <!-- empty appendix for now. It will up update in another issue-->
+                        <input class="form-control" list="Packaged/individual" name="Packaged-individual" id="Packaged-individual">
+                        <datalist id="Packaged/individual">
+                            <option value="Lab Package A">
```

Comment:
Why, are all the options, hard coded? This should be pulled from the database

Reply:
OK, I will work on that next week.
I need TA's help on that. 
I think I am missing something on pulling data from the database part.

Reply:
We could leave the form as is for now as a placeholder while you work on the functionality.
For an example of pulling data from the database, you could look at the order history page.

Reply:
It can all be done through a view basically. You can probably copy the order history view function to get you started.